spec/support/integration_test_setup.rb

```diff
@@ -0,0 +1,12 @@
+RSpec.shared_context "integration test setup" do
+  let(:server) { FactoryGirl.create(:miq_server) }
+  let(:user) { FactoryGirl.create(:user) }
+  let(:token_service) { Api::UserTokenService.new(:base => {:module => "api", :name => "API"}) }
+  let(:token) { token_service.generate_token(user.userid, "api") }
+
+  before do
+    Tenant.create!(:use_config_for_attributes => false)
+    allow(MiqServer).to receive(:my_guid).and_return(server.guid)
+    MiqServer.my_server_clear_cache
+  end
+end
```

Comment:
Hmmm we don't actually want to do this. 

* I'd prefer if we try and stub out such things by default. There's a *lot* of stuff we could test that wouldn't need this amount overhead for every single spec. 
* A shared context is sorta weird to use as a setup hook like this. 

Let's address this later.